test/ModuleInterface/access-level-import-swiftinterfaces.swift

```diff
@@ -101,9 +164,76 @@ private import PrivateLib
 // CHECK-NOT: PrivateLib
 
 //--- MultiFilesA.swift
+
 public import PublicLib
+package import PackageLib
 private import InternalLib
+public import LibX
+@_exported public import LibY
+@_implementationOnly @_spi(Lib) import LibZ
+@_spiOnly public import LibS
+@_spiOnly public import LibT
+@_spi(Lib) public import LibU
+@_spi(Lib) public import LibV
 
 //--- MultiFilesB.swift
+
 internal import PublicLib
+internal import PackageLib
 internal import InternalLib
+package import LibX
+package import LibY
+@_implementationOnly import LibZ
+package import LibS
+internal import LibT
+package import LibU
+internal import LibV
+
+// CHECK-5-MUL-NOT: public
+// CHECK-5-MUL: import LibU
+// CHECK-5-MUL: import LibV
+// CHECK-5-MUL: import LibX
+// CHECK-5-MUL: @_exported import LibY
+// CHECK-5-MUL: import PublicLib
+
+// CHECK-6-MUL: public import LibU
+// CHECK-6-MUL: public import LibV
+// CHECK-6-MUL: public import LibX
+// CHECK-6-MUL: @_exported public import LibY
+// CHECK-6-MUL: public import PublicLib
+
+// CHECK-5-MUL-PKG: package import LibS
+// CHECK-5-MUL-PKG: @_spiOnly import LibT
+// CHECK-5-MUL-PKG: @_spi(Lib) import LibU
+// CHECK-5-MUL-PKG: @_spi(Lib) import LibV
+// CHECK-5-MUL-PKG: import LibX
+// CHECK-5-MUL-PKG: @_exported import LibY
+// CHECK-5-MUL-PKG: package import PackageLib
+// CHECK-5-MUL-PKG: import PublicLib
+
+// CHECK-6-MUL-PKG: package import LibS
+// CHECK-6-MUL-PKG: @_spiOnly public import LibT
+// CHECK-6-MUL-PKG: @_spi(Lib) public import LibU
+// CHECK-6-MUL-PKG: @_spi(Lib) public import LibV
+// CHECK-6-MUL-PKG: public import LibX
+// CHECK-6-MUL-PKG: @_exported public import LibY
+// CHECK-6-MUL-PKG: @_implementationOnly @_spi(Lib) public import LibZ
+// CHECK-6-MUL-PKG: package import PackageLib
+// CHECK-6-MUL-PKG: public import PublicLib
+
+// CHECK-5-MUL-PRV: @_spiOnly import LibS
```

Comment:
Should this line be printed at all in private interface if it's supposed to have `package import`? See L214.